docs/dev/developer-guides/building-on-zksync/videos.md

```diff
@@ -0,0 +1,20 @@
+# Developer videos
+
+### zkSync 2.0 for Builders - Alex Gluchowski
+
+This talk covers everything you need to know about zkSync 2.0, and the user, and developer experience perspectives when building on zkSync.
+
+<iframe src="https://www.youtube.com/embed/xd2siik0iBU" title="zSync 2.0 for Builders - Alex Gluchowski" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Remove the "autoplay;" or all videos would autoplay on load in some browsers. Also try adding a width or height to make this page look better